scripts/gen_gitlab_config.py

```diff
@@ -249,11 +249,11 @@ def gen_appsec_iast_packages() -> None:
         f.write(
             """
 appsec_iast_packages:
```

Comment:
does this need to be it's own function anymore? can probably just delete and I think suitespec will generate the correct riot job